quickstarts/etcd/config.yml

```diff
@@ -0,0 +1,54 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: etcd
+
+# Displayed in the UI (required)
+title: etcd
+
+# Long-form description of the quickstart (required)
+description: |
+ ## What is etcd?
+
+ Etcd is a strongly consistent, distributed key-value store that provides a reliable way to store data that needs to be accessed by a distributed system or cluster of machines. It gracefully handles leader elections during network partitions and can tolerate machine failure, even in the leader node
+
+ ## Quickstart details
+
+ This quickstart was built based on `etcd` metrics sent to New Relic using Prometheus remote write.  If using with the Prometheus OpenMetrics Integration, the dashboard widgets will require some modification.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  A dashboard for your etcd Prometheus metrics
```

Comment:
A short summary is always nice, but this does look a little empty on the page. You might want to consider more copy here.
<img width="1668" alt="Screen Shot 2022-09-01 at 10 48 48 AM" src="https://user-images.githubusercontent.com/70179303/187979759-bf933673-8240-4151-af0a-7568b7b7945f.png">

Reply:
I added a little bit more.